examples/reachable.myl

```diff
@@ -9,6 +9,6 @@ DO
                               EMIT Edge.dst AS addr]);
     Delta = DIFF(NewlyReachable, Reachable);
     Reachable = UNIONALL(Delta, Reachable);
-WHILE [*COUNTALL(Delta) > 0];
+WHILE [FROM COUNTALL(Delta) AS size EMIT *size > 0];
```

Comment:
Yes, that's correct.  Basically, the star operator just does an implicit addition to the FROM clause, assuming the relation hasn't already been imported.  The star is idempotent for the reasons you describe.